src/mono/wasm/runtime/loader/run.ts

```diff
@@ -404,7 +404,13 @@ function initializeModules(es6Modules: [RuntimeModuleExportsInternal, NativeModu
 async function createEmscriptenMain(): Promise<RuntimeAPI> {
     if (!module.configSrc && (!module.config || Object.keys(module.config).length === 0 || !module.config.assets)) {
         // if config file location nor assets are provided
-        module.configSrc = "./mono-config.json";
+        if (loaderHelpers.scriptDirectory.indexOf("/_framework") == -1) {
```

Comment:
is this the other way around ? i'm confused about it

Reply:
`loaderHelpers.scriptDirectory` is absolute location of new `dotnet.js` loader.

Reply:
"If scriptDirectory doesn't contain `_framework`, use path with `_framework`"